integration/test/test.ts

```diff
@@ -90,7 +90,7 @@ declare type RunCallback =
     (err: Error|null, stdout?: string, stderr?: string) => void;
 
 function run(
-    cmd: string, args: string[], options: {[key: string]: any},
+    cmd: string, args: string[], options: {[key: string]: {}},
```

Comment:
why would we change `any` to an empty object.

Reply:
`gts fix` changed this, and I think it is better.  `any` means don't do type checks while `{}` means do type checks but the value can be any object without any constraints on its structure.

Reply:
I guess the benefit is to exclude any primitive types. Then it is better than any.